docs/source/implementations/clustering.rst

```diff
@@ -9,7 +9,109 @@ This implementation is based on 'Detection of False Investment Strategies using
 https://papers.ssrn.com/sol3/papers.cfm?abstract_id=3167017
 
 
+Implementation
```

Comment:
Why did you save the images into the labeling folder? These are not labeling images. Please add them to the correct folder, if none exists please create one.

Reply:
hello,

I have made a new folder called 'formula_images' and put the necessary PNG files to the folder and deleted the same PNG files from the folder of 'labeling_images'  - thanks